xarray/tests/test_backends.py

```diff
@@ -762,6 +762,18 @@ def test_0dimensional_variable(self):
                 expected = Dataset({'x': ((), 123)})
                 self.assertDatasetIdentical(expected, ds)
 
+    def test_read_open_dataset(self):
```

Comment:
For more clarity: `test_already_open_dataset`

Reply:
Done.